modules/administration_manual/nav.adoc

```diff
@@ -109,9 +96,23 @@
 **** xref:configuration/user/user_configuration.adoc[User Configuration]
 **** xref:configuration/user/user_provisioning_api.adoc[User Provisioning API]
 **** xref:configuration/user/user_roles.adoc[User Roles]
-** Upgrading
-*** xref:upgrading/marketplace_apps.adoc[Marketplace Apps]
-*** xref:upgrading/upgrade_php.adoc[Upgrading PHP]
+
+** xref:maintenance/index.adoc[Maintenance]
+*** xref:maintenance/upgrade.adoc[Upgrading]
+**** xref:maintenance/manual_upgrade.adoc[Manual Upgrade]
+**** xref:maintenance/package_upgrade.adoc[Upgrading from Package]
+**** xref:maintenance/update.adoc[Using the Updater App]
+**** xref:upgrading/upgrade_php.adoc[Upgrading PHP]
+**** xref:upgrading/marketplace_apps.adoc[Marketplace Apps]
+*** xref:maintenance/backup.adoc[Backup]
+*** xref:maintenance/enable_maintenance.adoc[Enable Maintenance]
+*** xref:maintenance/export_import_instance_data.adoc[Export and Import Instance Data]
+*** xref:maintenance/manually-moving-data-folders.adoc[Manually Moving Data Folders]
+*** Encryption
```

Comment:
Proposal fro the headline:
`Encryption related migration`

Reply:
I don't see the need to extend it, if it's a sub-section of Migration.